index.html

```diff
@@ -10,6 +10,11 @@
 </head>
 <body>
     <section class="mainpage">
+        <header class="toolbardesk">
+            <a href="#works">Portfolio</a>
+            <a href="#myself">About</a>
+            <a href="#contact">Contact</a>
+        </header>
```

Comment:
Great job @IngDavidTM 🤝

- [ ] Kindly make sure your design is a close replica of the design provided in the Figma template. You may please add that missing button as shown in the snapshot below.
- Your design
![Your design](https://user-images.githubusercontent.com/63917725/186514625-cc468523-4a3c-4f84-90e6-b3b4f2e9c46c.PNG)
- Figma design
![Figma design](https://user-images.githubusercontent.com/63917725/186514670-eda82e9e-2aee-4053-9190-cebcde4a51c0.PNG)